lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.scss

```diff
@@ -1,3 +1,7 @@
+[hidden] {
```

Comment:
Moving it to top of scss file will cause probably that all places will be affected with these styles because we have none encapsulation for that component. 

In theory maybe it has sense that hidden causes that element is hidden but then if we really want to make it global style then it should be in some file with global styles. In other case better to have it nested because it may be strange that styles for metadata component affect on all places.

Reply:
Yeah that might interfere sometimes, for now nested it locally